MapboxNavigation/NavigationCustomizable.swift

```diff
@@ -5,10 +5,6 @@ import MapboxCoreNavigation
  The `NavigationCustomizable` protocol represents a UI-based mechanism that allows for customization of its visual style, as well as the navigation service that powers it.
  */
 protocol NavigationCustomizable {
-    /**
-     The styles that the user interface can select from for display.
-    */
-    var styles: [Style]? { get }
```

Comment:
@JThramer what was the intention with this prop?

Reply:
@frederoni If I remember correctly the intention was to create a protocol that captured the similarities between the iOS NVC and the CPNVC. It's not really used for anything right now and can be removed.